.circleci/config.yml

```diff
@@ -166,9 +166,10 @@ test-chrome: &test-chrome
         command: |
           # --no-sandbox becasue we are running as root and chrome requires
```

Comment:
running as root? 😱

Reply:
Hmm I think we ran as root in travis, but don't in circle. So maybe that's unnecessary, I can look into it later.

Reply:
Turns out we do run in root. We don't request it, so I guess all circleCI jobs do? And that chrome bug is still present so we can't remove the no-sandbox flag...